contrib/devtools/lint-includes.sh

```diff
@@ -19,17 +19,6 @@ for HEADER_FILE in $(filter_suffix h); do
         echo
         EXIT_CODE=1
     fi
-    CPP_FILE=${HEADER_FILE/%\.h/.cpp}
```

Comment:
Possible followup (for a future pr) could be merge h/cpp loops since the files are treated the same now.